### ISSUE TITLE
Schema.org markup out of date

### ISSUE THREAD

**Issue by jdruiter** [woocommerce/woocommerce#12390]
## EXPLANATION OF THE ISSUE

Issue on the latest Woocommerce version.

Woocommerce produces Schema.org markup for product reviews, Like the following: 

```	
<div itemprop="reviewRating" itemscope="" itemtype="http://schema.org/Rating" class="star-rating" title="Rated 5 out of 5">
	<span style="width:100%"><strong itemprop="ratingValue">5</strong> out of 5</span>
</div>
```

Google's Structured Data Tool indicates the following problems: 
* The property reviewRating is not recognised by Google for an object of type Product.            
* Multiple reviews should accompanied by an Aggregatevreview

A more extended Schema.org markup would be preferred as well. Full example from Google:

```
<div itemscope itemtype="http://schema.org/Product">
  <span itemprop="name">Kenmore White 17" Microwave</span>
  <img itemprop="image" src="kenmore-microwave-17in.jpg" alt='Kenmore 17" Microwave' />
  <div itemprop="aggregateRating"
    itemscope itemtype="http://schema.org/AggregateRating">
   Rated <span itemprop="ratingValue">3.5</span>/5
   based on <span itemprop="reviewCount">11</span> customer reviews
  </div>
  <div itemprop="offers" itemscope itemtype="http://schema.org/Offer">
    <!--price is 1000, a number, with locale-specific thousands separator
    and decimal mark, and the $ character is marked up with the
    machine-readable code "USD" -->
    <span itemprop="priceCurrency" content="USD">$</span><span
          itemprop="price" content="1000.00">1,000.00</span>
    <link itemprop="availability" href="http://schema.org/InStock" />In stock
  </div>
  Product description:
  <span itemprop="description">0.7 cubic feet countertop microwave.
  Has six preset cooking categories and convenience features like
  Add-A-Minute and Child Lock.</span>
  Customer reviews:
  <div itemprop="review" itemscope itemtype="http://schema.org/Review">
    <span itemprop="name">Not a happy camper</span> -
    by <span itemprop="author">Ellie</span>,
    <meta itemprop="datePublished" content="2011-04-01">April 1, 2011
    <div itemprop="reviewRating" itemscope itemtype="http://schema.org/Rating">
      <meta itemprop="worstRating" content = "1">
      <span itemprop="ratingValue">1</span>/
      <span itemprop="bestRating">5</span>stars
    </div>
    <span itemprop="description">The lamp burned out and now I have to replace
    it. </span>
  </div>
  <div itemprop="review" itemscope itemtype="http://schema.org/Review">
    <span itemprop="name">Value purchase</span> -
    by <span itemprop="author">Lucas</span>,
    <meta itemprop="datePublished" content="2011-03-25">March 25, 2011
    <div itemprop="reviewRating" itemscope itemtype="http://schema.org/Rating">
      <meta itemprop="worstRating" content = "1"/>
      <span itemprop="ratingValue">4</span>/
      <span itemprop="bestRating">5</span>stars
    </div>
    <span itemprop="description">Great microwave for the price. It is small and
    fits in my apartment.</span>
  </div>

</div>
```

## STEPS TO REPRODUCE THE ISSUE

1. Install woocommerce
2. Leave a review
3. Check the rich snippet information on the review

**Comment by opportus** [comment]
Hi,

I presume you speak about the last **stable** WC version...

> A more extended Schema.org markup would be preferred as well. Full example from Google:

WC 2.7 is using now JSON-LD format and gives the possibility to extend further structured data via filter hooks. See #11455.

**Comment by claudiosanches** [comment]
I as just trying to find this reference @opportus, thanks.

@jdruiter it's already fixed for WooCommerce 2.7.

**Comment by jdruiter** [comment]
Thank you guys!

Quick and honest
